lib/models/loading_base.dart

```diff
@@ -62,9 +62,9 @@ class LoadingBase<T extends DataModel> extends LoadingMoreBase<T> {
         addAll(response.models!);
         total = response.total!;
         page = response.page ?? newPage;
-        lastId = cursorType == CursorType.json
-            ? (_decodeCursor(response.cursor)?['v'])
-            : response.cursor;
+        lastId = getCursorType?.call() != CursorType.json
```

Comment:
Use straight conditions as much as possible.

```suggestion
        lastId = getCursorType?.call() == CursorType.json
```

Reply:
```(getCursorType?.call() ?? CursorType.json ) == CursorType.json```
那应该要这样写了